recipes-robot/fs-overlay/ot3-overlay-mount-points_1.0.bb

```diff
@@ -0,0 +1,18 @@
+SUMMARY = "ot3 overlay mount point directory structure"
+LICENSE = "CLOSED"
+BB_STRICT_CHECKSUM = "0"
+PV = "1.0"
+
+S = "${WORKDIR}"
+
+inherit allarch
+
+do_install () {
+        install -d ${D}/data/upper/
+        install -d ${D}/data/upper/etc/
```

Comment:
oh and one minor thing, can we make the workdir hidden as `/data/.work` or something